Panic on unsupported operation: automatic sidecar injection

#### Current Behavior
Panicked when "Automatic sidecar injection" operation was run from Meshery UI.

```
➜  meshery-app-mesh git:(master) make run
DEBUG=true go run main.go
INFO[2021-10-18T18:19:30-05:00] Adaptor Listening at port: 10005              app=app-mesh-adapter
INFO[2021-10-18T18:22:11-05:00] Creating instance                             app=app-mesh-adapter
INFO[2021-10-18T18:22:11-05:00] Listing Operations                            app=app-mesh-adapter
INFO[2021-10-18T18:22:11-05:00] Creating instance                             app=app-mesh-adapter
INFO[2021-10-18T18:22:17-05:00] Creating instance                             app=app-mesh-adapter
INFO[2021-10-18T18:23:03-05:00] Creating instance                             app=app-mesh-adapter
INFO[2021-10-18T18:23:03-05:00] Applying operation label-namespace            app=app-mesh-adapter
panic: interface conversion: error is *errors.StatusError, not *errors.Error

goroutine 33 [running]:
github.com/layer5io/meshkit/errors.GetCode(0x5fbd600, 0xc000796a00, 0x0, 0x0)
	/Users/lee/code/go/pkg/mod/github.com/layer5io/meshkit@v0.2.31/errors/errors.go:90 +0xd1
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc000134088, 0x5fbd600, 0xc000796a00)
	/Users/lee/code/go/pkg/mod/github.com/layer5io/meshkit@v0.2.31/logger/logger.go:57 +0xaa
github.com/layer5io/meshery-adapter-library/adapter.(*Adapter).StreamErr(...)
	/Users/lee/code/go/pkg/mod/github.com/layer5io/meshery-adapter-library@v0.1.24/adapter/stream.go:25
github.com/layer5io/meshery-app-mesh/app_mesh.(*AppMesh).ApplyOperation.func3(0xc0005a6060, 0xf, 0xc0005a6070, 0xe, 0xc0005a6058, 0x8, 0x0, 0x0, 0x0, 0xc0005dc030, ...)
	/Users/lee/code/meshery-app-mesh/app_mesh/app_mesh.go:94 +0x199
created by github.com/layer5io/meshery-app-mesh/app_mesh.(*AppMesh).ApplyOperation
	/Users/lee/code/meshery-app-mesh/app_mesh/app_mesh.go:85 +0x565
exit status 2
make: *** [run] Error 1
```

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/app_mesh/error.go
+++ b/app_mesh/error.go
@@ -17,6 +17,9 @@ var (
 	ErrStreamEventCode = "appmesh_test_code"
 	// ErrSampleAppCode    failure
 	ErrSampleAppCode = "appmesh_test_code"
+	// ErrLoadNamespaceToMeshCode represents the error
+	// which is generated when the namespace could not be labelled and updated
+	ErrLoadNamespaceToMeshCode = "appmesh_test_code"
 	// ErrOpInvalidCode failure
 	ErrOpInvalidCode = "appmesh_test_code"
 	// ErrNilClientCode represents the error code which is
@@ -95,4 +98,8 @@ func ErrAppMeshCoreComponentFail(err error) error {
 // ErrProcessOAM is a generic error which is thrown when an OAM operations fails
 func ErrProcessOAM(err error) error {
 	return errors.New(ErrProcessOAMCode, errors.Alert, []string{"error performing OAM operations"}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrLoadNamespaceToMesh(err error) error {
+	return errors.New(ErrLoadNamespaceToMeshCode, errors.Alert, []string{"Could not label the appropriate namespace"}, []string{err.Error()}, []string{}, []string{})
 }

--- a/app_mesh/sample_apps.go
+++ b/app_mesh/sample_apps.go
@@ -59,7 +59,7 @@ func (appMesh *AppMesh) installSampleApp(namespace string, del bool, templates [
 func (appMesh *AppMesh) LoadNamespaceToMesh(namespace string, remove bool) error {
 	ns, err := appMesh.KubeClient.CoreV1().Namespaces().Get(context.TODO(), namespace, metav1.GetOptions{})
 	if err != nil {
-		return err
+		return ErrLoadNamespaceToMesh(err)
 	}
 
 	if ns.ObjectMeta.Labels == nil {
@@ -74,7 +74,7 @@ func (appMesh *AppMesh) LoadNamespaceToMesh(namespace string, remove bool) error
 
 	_, err = appMesh.KubeClient.CoreV1().Namespaces().Update(context.TODO(), ns, metav1.UpdateOptions{})
 	if err != nil {
-		return err
+		return ErrLoadNamespaceToMesh(err)
 	}
 	return nil
 }

